zebra/debug.c

```diff
@@ -450,7 +450,7 @@ DEFUN (no_debug_zebra_dplane,
 
 DEFPY (debug_zebra_nexthop,
        debug_zebra_nexthop_cmd,
-       "[no$no] debug zebra nexthop [detail$detail]",
+       "[no$no] debug zebra nexthop [detailed$detail]",
```

Comment:
From development channel:

do we want debug XXX detailed or debug XXXX detail?
12:14
sharpd@eva:~/frr/doc/user$ sudo vtysh -c "find detail" | grep  debug | grep detailed | wc
     10      54     408
sharpd@eva:~/frr/doc/user$ sudo vtysh -c "find detail" | grep  debug | wc
     36     226    2328
sharpd@eva:~/frr/doc/user$ 
12:14
we are aproximately 26 detail and 10 detailed
12:14
anyone care enough to go one way or another?

equinox  3:40 PM
detail is used throughout (164 times), detailed only has 14 hits in the entire CLI
3:40
detail wins hands-down i say

Reply:
Just putting the "changes requested" marker here for the 4-character (2 more below) change ;)

Reply:
`detail` would be better than `detailed,` IMHO.